src/ResourceManager/Automation/Commands.Automation/Cmdlet/StartAzureAutomationDscCompilationJob.cs

```diff
@@ -64,7 +79,7 @@ protected override void AutomationProcessRecord()
                                               Resources.ConfigurationDataShouldNotBeInJobParameters, "-ConfigurationData"));
             }
 
-            job = this.AutomationClient.StartCompilationJob(this.ResourceGroupName, this.AutomationAccountName, this.ConfigurationName, this.Parameters, this.ConfigurationData);
+            job = this.AutomationClient.StartCompilationJob(this.ResourceGroupName, this.AutomationAccountName, this.ConfigurationName, this.Parameters, this.ConfigurationData, this.IncrementNodeConfigurationBuild);
```

Comment:
@vrdmr have you tested this? I believe you will need to pass `this.IncrementNodeConfigurationBuild.IsPresent` to get the `bool` value this method expects.